gradle-plugin/src/main/groovy/akka/grpc/gradle/AkkaGrpcPlugin.groovy

```diff
@@ -162,14 +162,17 @@ class AkkaGrpcPlugin implements Plugin<Project> {
 
         project.task("printProtocLogs") {
             doLast {
-                Files.lines(logFile).forEach { line ->
-                    if (line.startsWith("[info]")) logger.info(line.substring(7))
-                    else if (line.startsWith("[debug]")) logger.debug(line.substring(7))
-                    else if (line.startsWith("[warn]")) logger.warn(line.substring(6))
-                    else if (line.startsWith("[error]")) logger.error(line.substring(7))
+                if (Files.exists(logFile)) {
```

Comment:
It looks like when a subproject doesn't generate any sources based on the proto because of `NO-SOURCE`, it won't produce a logging file either.

The example project above didn't have *any* proto files, but typically it'd have some in `api` but not in `impl`. I think it's OK to just skip without further logging in `impl` in that case.